libs/plugins/src/dashboard/README.md

```
@@ -0,0 +1,190 @@
+new DashboardPlugin({
+  cdn: {
+    entrypoint: 'https://cdn.example.com/frontmcp-dashboard@1.0.0/index.js',
+  },
+});
+```
+
+## Architecture
+
+The dashboard uses a standalone app architecture:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)